lib/miq_expression.rb

```diff
@@ -625,10 +625,14 @@ def sql_supports_atom?(exp)
       # => TODO: support count of child relationship
       return false if exp[operator].key?("count")
 
-      return field_in_sql?(exp[operator]["field"])
+      return field_in_sql?(exp[operator]["field"]) && value_in_sql?(exp[operator]["value"])
     end
   end
 
+  def value_in_sql?(value)
+    !Field.valid_field?(value) || Field.parse(value).attribute_supported_by_sql?
```

Comment:
Aah, I would prefer not to have hardcoded logic around what can and can't be in a value. Just simply asking if it is supported in sql.
If we add support later for these custom columns in sql, then I would like this to start returning true (because `arel_supports_sql? => true` and this would just work.